client/components/orderable-list/payment-method.scss

```diff
@@ -0,0 +1,119 @@
+.payment-method {
```

Comment:
(minor) I see that the `payment-method` class is passed as a prop to the `PaymentMethod` component, but it's actually the `OrderableList` component that "grabs" the `className` prop from its children and renders it on the `ListItem`.
I feel that would be a bit of a "gotcha" - do you think there's a way to ensure that the `payment-method` class is rendered by the `PaymentMethod` component, so that it can be styled there? Maybe by adding a wrapping `div` on the `PaymentMethod` component, rather than having the `OrderableList` inspect the props of its children?

```
const PaymentMethod = () => {
	return (
		<div className="payment-method">
                     ...
                 </div>
	);
};
```

Reply:
Absolutely! I avoided adding a `<div>` not to introduce unnecessary DOM elements but this would be much cleaner.

However, the current implementation relies on drag handles and other items (payment method title, description, actions) being siblings to be able to set `order: N` on them. Do you have any ideas on how to solve this?